articles/libraries/auth0js/v9/migration-guide.md

```diff
@@ -0,0 +1,108 @@
+---
+section: libraries
+title: Auth0.js v8 to v9 Migration Guide
+description: How to migrate from auth0.js v8 to auth0.js v9
+toc: true
+---
+# Migration Guide: Auth0.js v8 to v9
+
+This document lists all the changes that you should be aware of when migrating between versions 8 and 9 of Auth0.js. It includes information on what is changing and why, details on new or deprecated features, and instructions on how you can migrate your implementation.
+
+If you are using Auth0.js with your custom login in embedded mode, you are encouraged to use the latest and more secure version of the library, but before you update your code, make sure that you have reviewed this document and made any necessary changes in your implementation. 
+
+If you have any questions or concerns, you can submit them using the [Support Center](${env.DOMAIN_URL_SUPPORT}), or directly through your account representative, if applicable. 
+
+## Cross Origin Authentication
+
+If your implementation uses the `loginWithCredentials` method, then in order to use v9, you need to have Cross Origin Authentication (COA) enabled. COA has some limitations which you should be aware of, and can be read about in the [Cross Origin Authentication](/cross-origin-authentication) documentation.
```

Comment:
Cross-origin authentication does no longer need to be enabled for clients. I'll be automatically available for first-party clients. I would keep the link to how to configure the other required settings though.